docs/README.md

```diff
@@ -32,7 +32,7 @@ The following are not in scope:
 - Remove SUID-root from [numerous binaries](https://github.com/secureblue/secureblue/blob/live/files/scripts/removesuid.sh) and replace functionality [using capabilities](https://github.com/secureblue/secureblue/blob/live/files/system/usr/bin/setcapsforunsuidbinaries)
```

Comment:
why is the readme also getting moved to the docs dir?

Reply:
README works in `/`, `docs/`, and `.github/` too

Reply:
@Malix-Labs oh sweet, didn't know